CourseUIKit/ViewController.swift

```diff
@@ -8,12 +8,135 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    // MARK: - Private Properties
+    private var greetingLabel: UILabel!
```

Comment:
```suggestion
    private let greetingLabel = UILabel()
```
объявляй сразу и делай константой